src/actors/build/stresses_actor.jl

```diff
@@ -152,6 +165,7 @@ function solve_1D_solid_mechanics!(
     R_oh_out::T;                           # : (float) major radius of outboard edge of OH coil, meters
     axial_stress_tf_avg::T=NaN,            # : (float) average axial stress in TF coil core legs, Pa (if nothing, use constant fraction of hoop stress)
     axial_stress_oh_avg::T=NaN,            # : (float) average axial stress in OH coil, Pa (if nothing, use constant fraction of hoop stress)
+    tf_nose_thickness::T=0.0,              # : (float), thickness of solid nose section of TF coil, normalized to TF coil thickness
```

Comment:
```suggestion
    tf_nose_thickness::T,              # : (float), thickness of solid nose section of TF coil, normalized to TF coil thickness
```

Reply:
tf_nose_thickness = 0 means that there is no TF nose. Maintains default case.

Reply:
yes, I got that. I would just remove the default value, essentially forcing the users of this function to explicitly take ownership of that parameter.

Reply:
It shouldn't. `solve_1D_solid_mechanics!` is only used in the `ActorStresses`. Any new run will have `nose_hfs_fraction` fraction defined.

Either way, this is just a minor stylistic comment, it's really not important.